c10/util/string_utils.h

```diff
@@ -22,6 +22,12 @@ inline int stoi(const std::string& str, std::size_t* pos = 0) {
   int n = 0;
   ss << str;
   ss >> n;
+  if (ss.fail()) {
+    // To mimic `std::stoi` and to avoid including `Exception.h`, throw
+    // `std::invalid_argument`.
+    // We can't easily detect out-of-range, so we don't use `std::out_of_range`.
+    throw std::invalid_argument("Not an integer");
```

Comment:
I'm getting "error: no member named 'invalid_argument' in namespace 'std'
    throw std::invalid_argument("Not an integer");" on some of the internal builds. As far as I can tell we don't need to throw nice errors here, so do you think it would be fine to remove these?

Alternatively we could try to include <stdexcept> but I'm not really sure what the downstream ramifications of that are

Reply:
Let's try including `stdexcept`! Without this check `stoi(empty_string)` would succeed.